### ISSUE TITLE
Debian 9/10 & Ubuntu 16.04/20.04 inappropriate ioctl for device warnings

### ISSUE THREAD

**Issue by willprice** [GetPoplog/Seed#80]
Following the installation instructions to install poplog OBS results in the following error on Debian 9/10 and Ubuntu 16.04 and 20.04 (in a docker container):

```
$ poplog
;;; Warning: can't open shared object libXt.so (Inappropriate ioctl for
;;;     device)
;;; PRINT DOING
;;; DOING    :
;;; Warning: can't open shared object libXt.so (Inappropriate ioctl for
;;;     device)
;;; PRINT DOING
;;; DOING    :
;;; Warning: can't open shared object libX11.so (Inappropriate ioctl for
;;;     device)
;;; PRINT DOING
;;; DOING    :
;;; Warning: can't open shared object libX11.so (Inappropriate ioctl for
;;;     device)
;;; PRINT DOING
;;; DOING    :
Sussex Poplog (Version 16.0001 Mon Aug  2 22:16:17 UTC 2021)
Copyright (c) 1982-1999 University of Sussex. All rights reserved.

Setpop
:
```

**Comment by willprice** [comment]
These errors can be resolved by installing `libxt-dev` and `libx11-dev`

**Comment by willprice** [comment]
Resolved by #81
